lidar_camera_calibration_py/src/calibration_math.py

```
@@ -0,0 +1,49 @@
+            - R_est (np.ndarray): Estimated 3x3 rotation matrix.
+            - t_est (np.ndarray): Estimated 3x1 translation vector.
+    """
+    assert len(camera_points) == len(lidar_points)
+    camera_points = np.asarray(camera_points)
+    lidar_points = np.asarray(lidar_points)
+    centroid_camera = np.mean(camera_points, axis=0)
+    centroid_lidar = np.mean(lidar_points, axis=0)
+    camera_centered = camera_points - centroid_camera
+    lidar_centered = lidar_points - centroid_lidar
+    H = lidar_centered.T @ camera_centered
+    U, S, Vt = np.linalg.svd(H)
```

Ruff (0.14.0)

Unpacked variable `S` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)